oc-chef-pedant/spec/api/oc_id_spec.rb

```diff
@@ -51,7 +51,7 @@
 
   def signin(with_callback = true)
     response = get("#{platform.server}/id/signin", oc_id_user, headers: {"Accept" => "text/html"})
-    cookie = response.headers[:set_cookie][1].split(";").first
+    cookie = response.headers[:set_cookie][0].split(";").first
```

Comment:
a comment about the change with respect to the rails version would be useful here.